rero_ils/modules/loans/api.py

```diff
@@ -337,11 +337,14 @@ def location_by_pid(location_pid):
 
             :param location_pid: the location pid.
             """
+            fields = ['pid', 'name', 'library', 'pickup_name']
             if location_pid not in locations:
-                locations[location_pid] = LocationsSearch()\
+                data = LocationsSearch()\
                     .filter('term', pid=location_pid)\
-                    .source(includes=['pid', 'name', 'library'])\
+                    .source(includes=fields)\
                     .execute()[0].to_dict()
+                data = {k: v for k, v in data.items() if v}
```

Comment:
Why do we have empty fields at all?

Reply:
it' possible to have a `pickup_name` to None. Using, this, I removed this None value and UI will use library_name and location_name to display the pickup destinatioono